mavlink_vehicles/mavlink_vehicles.cc

```diff
@@ -79,6 +86,82 @@ bool is_same_socket(sockaddr_storage &r1, sockaddr_storage &r2)
 namespace mavlink_vehicles
 {
 
+namespace math
+{
+
+inline double rad2deg(double x)
+{
+    return (180.0 / M_PI) * x;
+}
+
+inline double deg2rad(double x)
+{
+    return (M_PI / 180.0) * x;
+}
+
+local_pos global_to_local_ned(global_pos_int point, global_pos_int reference)
+{
+    // Scaling factor to convert from 1e-7 degrees to meters at equator
+    // Given by: (M_PI/180)*EARTH_RADIUS_AT_EQUATOR
+    const double location_scaling_factor = 0.011131884502145034f;
+    double longitude_scaling_factor =
+        cosf(math::deg2rad(reference.lat * 1.0e-7f));
+
+    // Subtract and scale
+    local_pos local;
+    local.x = (point.lat - reference.lat) * location_scaling_factor;
+    local.y = (point.lon - reference.lon) * location_scaling_factor *
+              longitude_scaling_factor;
+    local.z = (point.alt - reference.alt) / 1000.0;
```

Comment:
Fixed in 6d6fc6cae4d3812ddd873a622ca3c27a617aa905. We still need to add comments to the .hh to properly define the expected unities for global_pos_int and local_pos_ned variables. But I think it's better to have it in the documentation patch.